src/Appium.Net/Appium/Service/AppiumLocalService.cs

```diff
@@ -32,6 +37,7 @@ public class AppiumLocalService : ICommandServer
         private readonly int Port;
         private readonly TimeSpan InitializationTimeout;
         private readonly IDictionary<string, string> EnvironmentForProcess;
+        private static readonly HttpClient SharedHttpClient;
```

Comment:
I can Dispose it once I call `DestroyProcess`

Reply:
Disposing it at the `DestroyProcess` or `Dispose` methods, is causing the tests to fail since we use the same AppiumLocalService across all tests. Do you have another suggestion on a safe way to dispose of it?

Reply:
Passing it through the ctor will be a Breaking change I would like to avoid.
What I can do is init the httpClient under the `internal AppiumLocalService`
something like that:
```
        internal AppiumLocalService(
            FileInfo nodeJS,
            string arguments,
            IPAddress ip,
            int port,
            TimeSpan initializationTimeout,
            IDictionary<string, string> environmentForProcess)
        {
            NodeJS = nodeJS;
            IP = ip;
            Arguments = arguments;
            Port = port;
            InitializationTimeout = initializationTimeout;
            EnvironmentForProcess = environmentForProcess;
            SharedHttpClient = CreateHttpClientInstance();
        }
```